blockchain/tx_pool.go

```diff
@@ -1761,6 +1782,9 @@ func (t *txLookup) Add(tx *types.Transaction) {
 	t.lock.Lock()
 	defer t.lock.Unlock()
 
+	t.slots += numSlots(tx)
+	slotsGauge.Update(int64(t.slots))
```

Comment:
Can we update `slotsGauge` elsewhere? Preferrably after `pool.all.Add()` and `pool.all.Remove()`.
- I see that there is only one `txLookup` instance under normal circumstances (since `TxPool.all == TxPool.priced.all`). But updating a global counter inside a container's method seems odd to me.
- Other counters such as `pendingReplaceCounter` are updated in `TxPool` methods.

Reply:
If the update is done in `Txpool` methods, it might done in too many places and i think it could leads to a mistake such as omitting if the code changes. Even though it's a global counter, its scope seems restricted to the `txLookup.slots`. In my opinion, i think it's okay to update in `txLookup` methods which writes the `txLookup.slots`. 

Btw, i have updated the code since the initialization is missed when initializing txLookup. Could you take a look? https://github.com/klaytn/klaytn/pull/1733/commits/8ad2cc49608b4038a1a00653ad4e3a96de06b9ea

Reply:
That's fair enough. I'll be okay with a comment somewhere saying that `txLookup` is effectively a singleton.